translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+        Perform a move that lets you gain two tiles.</source>
+        <target>Quand une tuile est quittée, elle devient potentiellement enlevable du plateau.
+        Pour qu&apos;elle soit enlevée, il faut qu'au moins trois de ses bords soient libres, et qu&apos;ils soient l&apos;un à côté de l&apos;autre.
+        Notez que si une tuile vide, voisine d&apos;une tuile qu&apos;on vient de retirer, devient retirable, elle sera retirée.
+        Par exemple, ci-dessous, en quittant sa tuile le pion foncé le plus haut ne déconnectera pas celle-ci !
+        Mais en quittant la tuile en bas à gauche, deux tuiles seront enlevées.&lt;br/&gt;&lt;br/&gt;
+        Effectuez un mouvement pour récupérer deux tuiles.</target>
+      </trans-unit>
+      <trans-unit id="4051956669507633461" datatype="html">
+        <source>Failed, you have not gained the two tiles that you could!</source>
+        <target>Raté, vous n'avez pas récupérer les deux tuiles que vous pouviez !</target>
+      </trans-unit>
+      <trans-unit id="1625619525907045191" datatype="html">
+        <source>Exchanging a tile</source>
+        <target>Échanger une tuile</target>
+      </trans-unit>
+      <trans-unit id="2710648987338283557" datatype="html">
+        <source>When you have a tile, you can see it on the left of the board.
+        As soon as you have two, you can click on an opponent&apos;s piece to capture it immediately.
+        This will cost you two tiles.
+        If any tiles are removed during that turn, nobody will get them.&lt;br/&gt;&lt;br/&gt;
+        Gain some time and capture the last piece!</source>
+        <target>Dès que vous avez au moins une tuile, vous pourrez le voir sur la gauche du plateau.
+        Dès que vous en avez deux, vous pouvez, en cliquant sur une pièce adverse, la capturer immédiatement.
+        Cet action vous coûtera deux tuiles.
+        Si une ou plusieurs tuile sont retirées pendant ce tour, personne ne les récupérera.&lt;br/&gt;&lt;br/&gt;
+        Gagnez du temps, et capturez la dernière pièce adverse !</target>
+      </trans-unit>
+      <trans-unit id="6149833006202189547" datatype="html">
+        <source>It&apos;s nice to move a piece, but you could have had the opponent&apos;s piece immediately by clicking on it!</source>
+        <target>C&apos;est bien gentil de se déplacer mais en cliquant sur la pièce vous l&apos;aurez immédiatement !</target>
+      </trans-unit>
+      <trans-unit id="4449916170244566677" datatype="html">
+        <source>Special capture</source>
+        <target>Capture spéciale</target>
+      </trans-unit>
+      <trans-unit id="794572598630804015" datatype="html">
+        <source>When a tile is removed from the board during your turn, some of the opponent&apos;s pieces may have lost their last free neighboring space, they will therefore be captured!
+        If this happens to one of your piece during your turn, it would however stay on the board.&lt;br/&gt;&lt;br/&gt;
```

Comment:
Intéressant, la distinction etnre "if ..., it will" (condition possible, à propos du présent ou futur) et "if ..., it would" (condition improbable, à propos du présent).
Donc du coup, wi, ouill!